resolwe_bio/processes/variant_calling/gatk4_hc.py

```diff
@@ -90,6 +89,12 @@ class Input:
         class Advanced:
             """Advanced options."""
 
+            interval_padding = IntegerField(
+                label="Interval padding",
+                default=100,
+                description="Amount of padding (in bp) to add to each interval "
```

Comment:
Perhaps also indicate in the description that this is relevant only when including the bed file.

Reply:
I made a change so this field will be hidden unless `intervals_bed` file is selected.